packages/react-components/react-tags-preview/stories/InteractionTag/InteractionTagDisabled.stories.tsx

```diff
@@ -17,19 +17,19 @@ export const Disabled = () => {
         <InteractionTagPrimary secondaryText="appearance=filled" icon={<CalendarMonthRegular />} hasSecondaryAction>
           disabled
         </InteractionTagPrimary>
-        <InteractionTagSecondary />
+        <InteractionTagSecondary aria-label="filled disabled, remove" />
       </InteractionTag>
       <InteractionTag disabled appearance="outline">
         <InteractionTagPrimary secondaryText="appearance=outline" icon={<CalendarMonthRegular />} hasSecondaryAction>
           disabled
         </InteractionTagPrimary>
-        <InteractionTagSecondary />
+        <InteractionTagSecondary aria-label="outline disabled, remove" />
       </InteractionTag>
       <InteractionTag disabled appearance="brand">
         <InteractionTagPrimary secondaryText="appearance=brand" icon={<CalendarMonthRegular />} hasSecondaryAction>
           disabled
         </InteractionTagPrimary>
-        <InteractionTagSecondary />
+        <InteractionTagSecondary aria-label="brand disabled, remove" />
```

Comment:
```suggestion
        <InteractionTagSecondary aria-label=" remove" />
```

The differences in each of these examples is visual right? in that case we probaly don't need the appearances in label here. 

Also is `disabled` not obvious for the secondary action that we need to put a label for it? that seems kinda scary

Reply:
So Milan helped me did the static analysis of a11y and found out this example has no label. But we need the 2nd round - the screen reader test to know what's the correct label. I'm just putting something I think it's correct there. Let me update this to just `remove` and see if this is good enough on the screen reader tests

Reply:
Actually other stories has for example `<InteractionTagSecondary aria-label="filled, remove" />`. I'll update all stories to have just `<InteractionTagSecondary aria-label="remove" />`.